HTML/Currents.html

```diff
@@ -0,0 +1,133 @@
+                        
+                        <h3>Introduction</h3>
+                        <ul>
+                            <li>In order to solve this module, you will be tracking the positions of four colored boats as they travel down a tower.</li>
+                            <li>The tower consists of 3 floors, each containing a system of currents leading to a drain. For simplicity, the floors are represented as 5x5 grids with the currents represented by arrows.</li>
+                            <li>When a boat enters a space, it will follow the arrows, until it reaches a drain, and then it will fall to the next floor, and continue until it exits the third floor of the tower at the bottom.</li>
+                            <li>The solution to the module is the number of times each colored boat makes its respective move. To easily remember the moves you must track for each boat, they are printed on the buttons. (Red = Up, Green = Right, Blue = Down, and Yellow = Left) </li>
+                            <li>Use the colored buttons to change the values on the screen so they match the correct values for each boat, and then press the submit button. If you submit the incorrect answer for any of the four boat values you will get a strike. The module does not tell you which boat was wrong.</li>
+                            <li>You can also press the clear button (C) to reset all the counts to their starting values. 
+                        </ul>
+                        <h3>Step 1: Setup the tower</h3>
+                        <ol>
+                            <li>There are 3 floors in the tower. In order to determine the order of the floors, locate the floors in the table below, and use the edgework and the initial values on the screen to calculate their priority value (<strong>PV</strong>). The floors have a priority order of A-&gt;B-&gt;C which means if multiple floors have the same <strong>PV</strong> then they will stay in the order they are in this list. (Ex. APV = 21 BPV = 19 CPV = 21, B-&gt;A-&gt;C)</li>
+                            <li>Once you find the order, these floors will be stacked with highest <strong>PV</strong> on bottom, and lowest <strong>PV</strong> on top.</li>
+                            <li>Next, rotate the middle floor clockwise 90 degrees, and the bottom floor counterclockwise 90 degrees. The currents rotate with the floor so be careful! (ex. an up current becomes a right current if rotated clockwise)</li>
+                            <li>Now you're ready to place the boats!</li>
+                        </ol>
+                </div>
+                <div class="page-footer relative-footer">Page 1 of 4</div>
+        </div>
+            <div class="page page-bg-02">
```

Comment:
These two should be on the same level, make sure to adjust everything in the document. Everything below (besides the last few lines) and one space off. You can run it through an auto-formatter if you need.